src/app.js

```diff
@@ -1,16 +1,29 @@
-const express = require("express");
+const LogStream = require("./myStream");
 const bodyParser = require("body-parser");
+const errorHandler = require("./error-handler");
+const express = require("express");
+const morgan = require("morgan");
 const repositoryCotroller = require("./controllers/repository");
+const userCotroller = require("./controllers/user");
+const { HttpNotFound } = require("./utils/errors");
```

Comment:
```suggestion
const { HttpNotFoundError } = require("./utils/errors");
```

Para erro legal deixar bem explicitor que eh um error. Para não ser confundido com outros tipos de classes.